crates/pallet-pot/src/lib.rs

```diff
@@ -202,3 +209,21 @@ impl<T: Config<I>, I: 'static> OnUnbalanced<NegativeImbalanceOf<T, I>> for Palle
         });
     }
 }
+
+/// A helper to handle `OnUnbalanced` implementation over `CreditOf`
+/// to avoid conflicting implmentation.
+pub struct OnUnbalancedOverCredit<T, I>(Pallet<T, I>);
+
+impl<T: Config<I>, I: 'static> OnUnbalanced<CreditOf<T, I>> for OnUnbalancedOverCredit<T, I> {
```

Comment:
```suggestion
impl<T: Config<I>, I: 'static> OnUnbalanced<CreditOf<T, I>> for Pallet<T, I> {
```

Reply:
https://github.com/humanode-network/humanode/pull/861/commits/a8472217e441eaba2e9b434848739ded4516d30c

Reply:
Well, the also need to update the runtime config accordingly. Also, this breaks clippy.

Reply:
Fixed